### PR TITLE
create footer button and title holder components

### PR DESCRIPTION
**Pull request by raihankabir36850**
In the product folder, I created the titleholder component and button component.
Inside the component folder, I created the footer component and its associated CSS property.
Now the first page of our e-commerce looks like
screencapture-localhost-3000-2021-01-24-18_35_50

### CODE REVIEW

File: src/components/product/TitleHolder.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Typography } from 'antd';
+import styles from './TitleHolder.module.css'
+
+const { Title } = Typography;
+
+const TitleHolder = ({ title }) => {
```
Comment (alamiiin) [inline src/components/product/TitleHolder.js]: Change TitleHolder with Header.

File: src/components/product/ProductList.js [hunk 2]
```diff
@@ -82,11 +81,7 @@ function ProductList() {
           </Col>
         ))}
       </Row>
-      <div className="view-products">
-        <Button type="link" className="my-button">
```
Comment (alamiiin) [inline src/components/product/ProductList.js:86]: my-button -> What is the function of this class name?

File: src/components/product/ProductList.js [hunk 3]
```diff
@@ -68,9 +69,7 @@ const itemList = [
 function ProductList() {
   return (
     <div className={styles.containerFluid}>
-      <div className="title-holder">
-        <h2>POPULAR PRODUCTS</h2>
```
Comment (alamiiin) [inline src/components/product/ProductList.js:72]: Use css to capitalize.

File: src/components/product/ProductList.js [hunk 4]
```diff
@@ -13,6 +12,8 @@ import image7 from "../../assets/images/denim-tshirt.jpg";
 import image8 from "../../assets/images/full-black.jpg";
 
 import CardItem from "./ProductCard";
+import TitleHolder from './TitleHolder';
```
Comment (alamiiin) [inline src/components/product/ProductList.js]: Header

File: src/components/Footer/Footer.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+} from '@ant-design/icons';
+import 'antd/dist/antd.css';
+import styles from './Footer.module.css';
+
+function Footer() {
+    return (
+        <div className={styles.footer}>
+            <Row className={styles.footerSectionOne}>
+                <Col span={8}>
+                    <p>Contact us</p>
```
Comment (alamiiin) [inline src/components/Footer/Footer.js]: These should be links not paragraph. Use `Link` from `react-router`.

File: src/components/product/TitleHolder.js (hunk 1 above)
Comment (raihankabir36850) [reply src/components/product/TitleHolder.js]: Done. check, please

**Review by alamiiin — APPROVED** [review]

**Pull request merged**